src/main/resources/application.properties

```diff
@@ -0,0 +1,66 @@
+mainWindowTitle=Downlord's FAF Client
+forgedAlliance.windowTitle=Forged Alliance
+
+irc.host=irc.faforever.com
+irc.port=8167
+irc.defaultChannel=#aeolus
+irc.reconnectDelay=5000
+lobby.host=test.faforever.com
+lobby.port=8001
+
+update.host=faforever.com
+update.port=9001
+
+stats.host=faforever.com
+stats.port=11002
+
+replay.host=lobby.faforever.com
+replay.port=11002
+
+turn.host=faforever.com
+turn.port=3478
+stun.host=faforever.com
+newsFeedUrl=http://direct.faforever.com/news/feed
+
+connectivity.helpUrl=http://wiki.faforever.com/index.php?title=Connection_issues_and_solutions
+
+vault.mapDownloadUrl=http://content.faforever.com/faf/vault/maps/%s.zip
+vault.mapPreviewUrlFormat=http://content.faforever.com/faf/vault/map_previews/%s/%s.png
+vault.replayDownloadUrl=http://replay.faforever.com/%d
+vault.modRoot=http://content.faforever.com/faf/vault/mods/
+
+playerInfo.nameHistoryUrl=http://app.faforever.com/faf/userName.php?displayName=%s
+
+fafExe.url=http://content.faforever.com/faf/updaterNew/updates_faf_files/ForgedAlliance.exe
+fafReplayServer.host=lobby.faforever.com
+fafReplayServer.port=15000
+localReplayServer.port=15000
+replayFileFormat=%d-%s.fafreplay
+replayFileGlob=*.fafreplay
+
+github.releases.url=https://api.github.com/repos/FAForever/downlords-faf-client/releases
+github.releases.timeout=5000
+
+imgur.upload.baseUrl=https://api.imgur.com/3/image
+imgur.upload.clientId=141ee8a7030f16d
+imgur.upload.maxSize=2097152
+
+rating.initialStandardDeviation=500
+rating.initialMean=1500
+rating.beta=250
+rating.low=500
+rating.moderate=1000
+rating.good=1500
+rating.high=2000
+rating.top=2500
+api.baseUrl=https://api.test.faforever.com
+oauth.loginUri=${api.baseUrl}/login
+oauth.authUri=${api.baseUrl}/oauth/authorize
+oauth.tokenUri=${api.baseUrl}/oauth/token
+oauth.redirectUris=http://localhost
+oauth.clientId=3bc8282c-7730-11e5-8bcf-feff819cdc9f
+oauth.clientSecret=6035bd78-7730-11e5-8bcf-feff819cdc9f
+unitDatabase.url=http://spooky.github.io/unitdb/#/
+
+login.createAccountUrl=https://www.faforever.com/account/register
+login.forgotLoginUrl=http://app.faforever.com/faf/forgotPass.php
```

Comment:
`application.properties` does no longer exist; you'll have to add these values to `application.yml`